### DISCUSSION TITLE
Please set a hostname for 127.0.0.1 in your /etc/hosts error while installing dokku.

### DISCUSSION THREAD

**Discussion by babarali539** [dokku/dokku#5572] [Q&A]
Hi all!
I am trying to install dokku on ubuntu 20.* but when I run
`sudo DOKKU_TAG=v0.12.5 bash bootstrap.sh`
I get the following error
"This installation script requires that you have a hostname set for the instance. Please set a hostname for 127.0.0.1 in your /etc/hosts"
I have also added the host in /etc/hosts and here is the content for this file
```
127.0.0.1	localhost
127.0.1.1	ahsan-Inspiron-5567
127.0.0.1       dokku.example.com

# The following lines are desirable for IPv6 capable hosts
::1     ip6-localhost ip6-loopback
fe00::0 ip6-localnet
ff00::0 ip6-mcastprefix
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters
```
I am unable to understand what I am doing wrong. Any help would be much appreciated.

**Comment by josegonzalez** [comment]
I think `hostname -f` needs to succeed. The error message we print might be wrong now. 

`hostnamectl` might be useful.

See this for more details: https://www.cyberciti.biz/faq/ubuntu-change-hostname-command/
